build.assets/Makefile

```diff
@@ -227,6 +232,24 @@ buildbox-teleterm: buildbox
 		--cache-from $(BUILDBOX_TELETERM) \
 		--tag $(BUILDBOX_TELETERM) -f Dockerfile-teleterm .
 
+
+CONNECT_VERSION ?= $(VERSION)
+ifeq ($(CONNECT_VERSION),)
+CONNECT_VERSION := $(BUILDBOX_VERSION)-dev
```

Comment:
I'm not sure about it, for the other artifacts we have a tag name:
<img width="403" alt="image" src="https://user-images.githubusercontent.com/20583051/185949833-8db77c9e-c68d-474b-b694-a6e68222d19c.png">